docker-images/artifacts/kafka-thirdparty-libs/4.0.x/pom.xml

```diff
@@ -28,6 +28,7 @@
         <opentelemetry.instrumentation.version>1.32.0-alpha</opentelemetry.instrumentation.version>
         <grpc-netty-shaded.version>1.61.0</grpc-netty-shaded.version>
         <guava.version>32.1.3-jre</guava.version>
+        <log4j.version>2.24.3</log4j.version>
```

Comment:
A question ... I noticed this version is different from the one used for the Operators and Cruise Control. Is it because Kafka 4.0 is using 2.24.3?

Reply:
Great, thanks, that is good. Maybe we should update it int eh operators as well -> but we should definitel leave it for a separate PR later.